test/css-stats.json

```diff
@@ -4,98 +4,114 @@
     "base64Length": 0,
     "redundantBodySelectors": 0,
     "redundantChildNodesSelectors": 0,
-    "colors": 18,
+    "colors": 32,
```

Comment:
Why is this file checked in?

Reply:
it keeps track of complexity in the output CSS, so we can see regressions. It was part of the CI setup too, but that needs to be re-setup after all the recent changes here